### PR TITLE
Feature/rso sample apps

### PR DESCRIPTION
**Pull request by extferhernandez**
PR including RSO Sample apps in Go, Nodejs, and Python

### CODE REVIEW

**Comment by geozukunft** [comment]
From [README.md](https://github.com/RiotGames/developer-relations/blob/main/README.md)

_Welcome to the Riot Games Developer Relations repo. The purpose of this repo is **exclusively bug reporting** for tools in the Riot Games Developer Ecosystem._

File: sample_apps/rso_sample_apps/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+```
+RSO_BASE_URL=https://auth.riotgames.com
+RSO_CLIENT_ID=riot-example-app
+RSO_CLIENT_SECRET=*******************************
+APP_BASE_URL=http://localhost.riotgames.com:3000
+APP_CALLBACK_PATH=/oauth-callback
+RESPONSE_TYPE=code
+SCOPE=openid
+```
+# Sequence diagram 🟦➡️🟩⬅️
```
Comment (lucasMesquitaBorges) [inline sample_apps/rso_sample_apps/README.md:48]: Comment out all the TDB until they're defined

File: sample_apps/rso_sample_apps/go/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Go Sample app
+
+## Requirements
+1. Make sure you have your `.env` in the root of the project: `samples/.env` not `samples/.env`
```
Comment (lucasMesquitaBorges) [inline sample_apps/rso_sample_apps/go/README.md:4]: The alternatives are the same

**Review by DarkIntaqt — CHANGES_REQUESTED** [review]
I would not push a pycache file

File: sample_apps/rso_sample_apps/nodejs/index.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+    html += `
+    </tbody>
+    </table>
+        `
+
+    return style + html;
+
+}
+app.listen(port, () => console.log(`Example app listening on post ${port} !`));
```
Comment (leonardodimarchi) [inline sample_apps/rso_sample_apps/nodejs/index.js:147]:
```suggestion
app.listen(port, () => console.log(`Example app listening on port ${port}!`));
```

File: sample_apps/rso_sample_apps/nodejs/package.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "version": "0.0.1",
+  "description": "rso example app",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Riot Tuxedo",
+  "license": "ISC",
+  "dependencies": {
```
Comment (leonardodimarchi) [inline sample_apps/rso_sample_apps/nodejs/package.json:12]: I think it would be cool to have the "engines" here, specifying the nodejs version

**Review by leonardodimarchi — CHANGES_REQUESTED** [review]

File: sample_apps/rso_sample_apps/go/README.md (hunk 2 above)
Comment (klahnen) [reply sample_apps/rso_sample_apps/go/README.md]: nice catch, done!

File: sample_apps/rso_sample_apps/README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* Get your secrets:
+    * RSO_BASE_URL=https://auth.riotgames.com
+    * RSO_CLIENT_ID=<your-client-id>
+    * RSO_CLIENT_SECRET=<your-client-secret>
+    * DEPLOYMENT_API_KEY=<your-deployment-api-key>
+    * APP_BASE_URL=http://<your-app-base-url>
+    * APP_CALLBACK_PATH=<your-callback-path>
+    * RESPONSE_TYPE=code
+    * SCOPE=openid
+* Make sure you have your `.env` in the root of the project: `samples/.env` not `samples/.env`
```
[the remark this answers is not in the document]
Comment (klahnen) [reply sample_apps/rso_sample_apps/README.md:23]: nice catch, is fixed now!

File: sample_apps/rso_sample_apps/README.md (hunk 1 above)
Comment (klahnen) [reply sample_apps/rso_sample_apps/README.md:48]: commented, thanks!

**Comment by lucasMesquitaBorges** [comment]
LGTM!

**Pull request merged**